charts/incubator/hyperswitch-cd/hyperswitch-app-argo-application.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
```

Comment:
how do we use this applicationset without chart.yaml?

Reply:
Added the chart.yml in hyperswitch-cd.